cqlsh_tests/test_cqlsh_types.py

```diff
@@ -0,0 +1,66 @@
+import time
+
+import logging
+
+from dtest import Tester, create_ks
+
+logger = logging.getLogger(__name__)
+
+
+class TestCqlshTypes(Tester):
+
+    def prepare(self, workload=None):
+        if not self.cluster.nodelist():
+            self.allow_log_errors = True
+            self.cluster.populate(1)
+            if workload is not None:
+                for node in self.cluster.nodelist():
+                    node.set_workload(workload)
+            logger.debug('About to start cluster')
+            self.cluster.start()
+            logger.debug('Cluster started')
+            for node in self.cluster.nodelist():
+                node.watch_log_for('Starting listening for CQL clients', timeout=60)
+            self.cluster.nodelist()[0].watch_log_for('Created default superuser')
+            self.node = self.cluster.nodelist()[0]
+
+            conn = self.patient_cql_connection(self.node)
+            create_ks(conn, 'ks', 1)
+
+        logger.debug('prepare completed')
+
+    def test_point(self):
```

Comment:
I think we need `@since("4.0")` or similar on these tests?

Reply:
yes, added